src/containers/Trigger/Trigger.test.tsx

```
@@ -1,207 +1,455 @@
 dayjs.extend(utc);
 
-vi.mock('react-router', async () => {
+const mockUseLocationValue: any = {
+  pathname: '/',
+  search: '',
+  hash: '',
+  state: null,
+};
+
+vi.mock('react-router', async () => ({
+  ...((await vi.importActual<any>('react-router')) as {}),
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)